docs/Modern-Mutations.md

```diff
@@ -37,7 +37,10 @@ commitMutation(
 * `config`:
   * `mutation`: The `graphql` tagged mutation query.
   * `variables`: Object containing the variables needed for the mutation. For example, if the mutation defines an `$input` variable, this object should contain an `input` key, whose shape must match the shape of the data expected by the mutation as defined by the GraphQL schema.
-  * `onCompleted`: Callback function executed when the request is completed and the in-memory Relay store is updated with the `updater` function. Takes a `response` object, which is the "raw" server response, and `errors`, an array containing any errors from the server. .
+  * `onCompleted`: Callback function executed when the request is completed and the in-memory Relay store is updated with the `updater` function. Takes a `response` object, which is the "raw" server response, and `errors`, an array containing any errors from the server. 
+**Please note:** 
+    * This callback (and not `onError`) will be executed whenever the response contains `data`, even if `data` is `null`.
```

Comment:
This partly depends on your network layer. If the network layer throws an exception, then you hit `onError`.

Reply:
@milosa mind adding that clarification?